src/annotator/range-util.js

```diff
@@ -10,6 +10,7 @@ export function isSelectionBackwards(selection) {
   }
 
   const range = selection.getRangeAt(0);
+  // Does not work in iOS because these two nodes are never equal.
   return range.startContainer === selection.focusNode;
```

Comment:
What was the relation between `range.startContainer` and `selection.focusNode` in your iOS testing? According to [the spec](https://w3c.github.io/selection-api/#dfn-focus):

> Each selections also have an anchor and a focus. If the selection's range is null, its anchor and focus are both null. If the selection's range is not null and its direction is forwards, its anchor is the range's start, and its focus is the end. Otherwise, its focus is the start and its anchor is the end.

So it seems from a reading of the spec that this check should be valid and if not, might indicate a bug that we should report.

Reply:
I need to get the exact objects so I can be more specific, its a bit hard to debug on my ipad, I need to find a better way.

Reply:
This comment was marked as resolved but I didn't get an answer on the query above. Did you figure out what happens here on iOS?